src/main/java/ladderrefactoring/domain/ladder/Line.java

```diff
@@ -0,0 +1,50 @@
+package ladderrefactoring.domain.ladder;
+
+import ladderrefactoring.strategy.DirectionStrategy;
+
+import java.util.*;
+
+public class Line {
+
+    public static final int SECOND_INDEX = 1;
+
+    private final List<Point> points = new ArrayList<>();
+
+    public Line(int widthOfLadder, DirectionStrategy directionStrategy) {
+        initLine(widthOfLadder, directionStrategy);
+    }
+
+    private void initLine(int widthOfLadder, DirectionStrategy directionStrategy) {
+        Point currentPoint = Point.first(directionStrategy);
+        points.add(currentPoint);
+
+        for (int i = SECOND_INDEX; i < widthOfLadder-1; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < widthOfLadder - 2; i++) {
```
이렇게 변경해보는 것은 어떨까요?
-2는 상수(OFFSET)로 선언해서 사용하고요.